### DISCUSSION TITLE
tutorials/voyage-part1/10-defining-an-entity

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#89] [course feedback]
# tutorials/voyage-part1/10-defining-an-entity

https://www.apollographql.com/tutorials/voyage-part1/10-defining-an-entity

**Comment by torressam333** [comment]
Why did we have to add a Location Type to the Reviews subgraph but we didn't add a type Review to the Locations subgraph schema? Thanks!

**Reply by lizhennessy** [reply]
Hi there @torressam333 ! We added the `Location` entity to the `reviews` subgraph because we have reviews-related data that we want to contribute as a field to the `Location` type: as you'll see in an upcoming lesson, we want to provide the actual reviews data for a particular location. This lets us run queries for multiple locations _and_ their corresponding reviews.

We certainly _could_ add a `Review` entity to the `locations` subgraph; we might do this if we had some location-specific data that we wanted to attach to each review object that we returned. But in our case, a particular location has a one-to-many relationship with its reviews; each review is attached to a _single_ location, but a single location can have _many_ reviews. For this reason, it makes more sense to boost the `Location` type with a field for its (potentially many) reviews!

**Reply by torressam333** [reply]
Awesome, that makes perfect sense. Thanks!
